### DISCUSSION TITLE
An error is No root methods

### DISCUSSION THREAD

**Discussion by wenmin864** [eclipse/windowbuilder#701] [Q&A] [answered]
when I open the Design Tab in a new plugin,  I get an error ," No root methods"
![1706493067935](https://github.com/eclipse-windowbuilder/windowbuilder/assets/51218435/aa07b60b-395e-4620-8c6d-b9dc2ea9ad76)

eclipse version is 2023-12 and WindowsBuilder version is 1.15
![17064931538784](https://github.com/eclipse-windowbuilder/windowbuilder/assets/51218435/7271dfe9-9434-46df-a12e-e72dcdf0b29d)

My testCode is sample , But it works fine when I use eclipse 2023-09
![17064933014248](https://github.com/eclipse-windowbuilder/windowbuilder/assets/51218435/2f45f4ae-7b21-4b0e-8752-7f55dbda29d5)

**Comment by ptziegler** [comment]
Are you using javax.annotation.PostConstruct or jakarta.annotation.PostConstruct?

**Reply by wenmin864** [reply]
I use  jakarta.annotation.PostConstruct.And eclipse 2023-12 can not  import package:javax.annotation.PostConstruct

**Reply by ptziegler** [reply, marked as answer]
> I use jakarta.annotation.PostConstruct

That's what I thought. Support for jakarta annotations was only added recently via #700.
Can you try it again with the M2?
https://download.eclipse.org/windowbuilder/updates/milestone/S202401292103

**Reply by wenmin864** [reply]
Thansk for your reply, WindowsBuilder is working fine.
